types/netvox/door/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Door R311A-Wireless Door/Window Sensor (LoRaWAN)",
+  "name": "R311A",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed